esx_service/utils/auth_data.py

```diff
@@ -336,6 +366,7 @@ def connect(self):
         
         if need_create_table:
             self.create_tables()
+            self.create_fallback_tenant_and_privileges()
```

Comment:
fallback has nothing to do with upgrade.
fallback tenant and fallback privilege will be created automatically when table was first created. 
upgrade happens when we detect that there exist a DB file, but we found the version is not the latest, then we should have some code to upgrade the DB table to the latest. Those part of code is not here yet.

Since user can remove the fallback tenant and fallback privilege, in unit test, we have some test to test the behavior when fallback tenant and fallback privileges are removed. And after our test, we want to return back to the status that fallback tenant and fallback privileges are still in the DB table.